config/eslintrc.template.js

```diff
@@ -77,6 +77,7 @@ module.exports = ({ tsconfigRootDir }) => ({
     'react/no-unused-class-component-methods': 'warn',
     'react/prop-types': 'off',
     'react/require-default-props': 'off',
+    curly: [2, 'all'],
```

Comment:
```suggestion
    'curly': [2, 'all'],
```

I think we should use strings as key.

Reply:
Breaks ESLint: `Unnecessarily quoted property 'curly' found`

Reply:
Interesting 😄